### DISCUSSION TITLE
Is ist possible to layout two tableviews on the left and right half of the view

### DISCUSSION THREAD

**Discussion by softwareengel** [serenity-is/Serenity#5517] [Help]
generating tableview ist working good, one table per view. But how can I place two tableviews beside one another (split plane horitontal, place one tableView left and one tableview on the right)? e.G. TableA and TableB - no relations but both with editable checkboxes.

see : https://stackoverflow.com/questions/66262057/is-ist-possible-to-layout-two-tableviews-on-the-left-and-right-half-of-the-view

**Comment by VictorTomaili** [comment]
There some samples similar. But they are included by premium packages. You need to write own scripts for this. 

https://serenity.is/demo/AdvancedSamples/SplitMasterDetail
https://serenity.is/demo/AdvancedSamples/EntityGridDialog#edit/1

**Reply by softwareengel** [reply]
Thanks, but that "similar" doesn't help me. Writing code is okay, but is there somthing like a split-pane - container for more than one View-Element? Or is in this serenity concept only one complex view element in the content area possible?

**Reply by VictorTomaili** [reply]
Similar samples means there is nothing for your specific requirements. 

There is only limit is yourself. Serenity just a platform. You can override, change, customize everything. Also it's a .net 5 application and you can to do everything what you can in .net 5 app
